Source/Resources/Content/MSGraph/Groups/BUILTIN - Microsoft 365 Users.json

```diff
@@ -2,7 +2,7 @@
   "assignedLicenses": {
     "$ref": "ResourceContext:M365Licenses"
   },
-  "description": "This dynamic group is used to assign Microsoft 365 licenses to users for EMS and O365 functionality.",
+  "description": "What is it: A dynamic group used to assign Microsoft 365 licenses to users for EMS and O365 functionality. \n Simeon Notes: All users that have a department will be included in this group. In addition to licensing, this group controls policies that should be applied to all users e.g., password reset.",
```

Comment:
**What does this do?**
Creates a dynamic group used to assign Microsoft 365 licenses to users for EMS and 0365 functionality. In addition to licensing, this group controls policies that should be applied to all users e.g., password reset. The baseline places users that have a department into this group. 

**Why should you use this?**
If you want to tighten security around who can assign licenses to users. 

**What is the end user impact?**
Users who are not in this group will be restricted from assigning licenses.

**Learn more**
NA